### DISCUSSION TITLE
Multilingual text templates

### DISCUSSION THREAD

**Discussion by moerphie** [orestbida/cookieconsent#228] [Ideas]
Hi there,
first of all: Thank you for this open source project, @orestbida. It's awesome.

In my opinion it would be very helpful to define text templates for the most common cookies (Google Analytics, Matomo, XSRF-Token, ..) – with the help of the community in different languages. 

This could be included into the code, or just in a markdown-file.

```javascript
// ...
cookie_table: [
    {
        col1: '_ga',
        col2: cookieconsent.getTextTemplate('en', '_ga', 'domain'),
        col3: cookieconsent.getTextTemplate('en', '_ga', 'description')',
    },
    {
        col1: '_gid',
        col2: cookieconsent.getTextTemplate('en', '_gid', 'domain'),
        col3: cookieconsent.getTextTemplate('en', '_gid', 'description')',
    }
]
// ...
```

**Comment by orestbida** [comment]
Hi @moerphie,

I don't dislike the idea, but in all honesty I don't think it's worth it especially if I consider that one has already gone through the "trouble" of configuring the plugin in multiple languages. 

My idea would have been different if there were specific pre-defined categories and translations for each cookie category.
